docs/modules/integrate/pages/kafka-connector.adoc

```diff
@@ -103,6 +103,35 @@ properties.put("schema.registry.url", schemaRegistryUrl);
 Keep in mind that once the record deserialized, Jet still needs to know
 how to serialize/deserialize the record internally.
 
+== Avro Schema
+To use an Avro schema for `GenericRecord`, you must provide the following as parameters for the Kafka source and/or sink:
+
+* JSON representation of Avro schema
+* A `HazelcastKafkaAvroSerializer` as key/value serializer class
+* A `HazelcastKafkaAvroDeserializer` as key/value deserializer class
+
+The process will utilize the schema from properties, disregarding the schema from the record. An exception will be raised if the types of the record schema and properties schema are different.
+
+The parameters can be specified as shown in the following examples.
+
+Example source parameters:
+```java
+properties.put("key.deserializer", HazelcastKafkaAvroDeserializer.class);
+properties.put("value.deserializer", HazelcastKafkaAvroDeserializer.class);
+properties.put("keyAvroSchema", "{"type":"record","name":"key","namespace":"schema","fields":[{"name":"key","type":["null","int"],"default":null}]}");
```

Comment:
JSON has to be escaped for valid Java

Reply:
ok
02a4a468b1027ab2cf0a3a88406397d755d09e5d